### PR TITLE
More comprehensive list of GW ingesting and querying use cases.

### PR DESCRIPTION
**Pull request by mses-bly**

### CODE REVIEW

**Comment by rnix** [comment]
- Please change the indenting to tabs.
- Please remove all the `++++++++>` styling to stdout. I don't see anything like that in their examples. Alternatively, switch it all to `log.info` and `log.debug` statements.
- Can you apply this PR to a fresh clone of the GeoWave repo then document the steps required to run the example? I'm getting a `ClassNotFoundException: org.apache.accumulo.core.client.security.tokens.AuthenticationToken` when i do exactly that in IntelliJ even though `mvn compile` succeeds. If it becomes more involved than two or three steps, it may be nice to have a Maven profile configured that'll document how to run the example.

**Comment by mses-bly** [comment]
I thought our Spatially/geowave fork was fresh. Let me try again on my side to see if I can reproduce the ClassNotFoundException.
*Update: I can avoid entirely the use of the class that is giving the error if I use a deprecated function call. It's not ideal, but potentially avoids doing dependency configs. Is that a desirable?
